designer/server/src/main/scala/pl/touk/nussknacker/ui/api/ScenarioActivityApiHttpService.scala

```diff
@@ -136,8 +116,9 @@ class ScenarioActivityApiHttpService(
 
   private def deleteComment(request: DeleteCommentRequest): EitherT[Future, ScenarioActivityError, Unit] =
     EitherT(
-      scenarioActivityRepository.deleteComment(request.commentId)
-    ).leftMap(_ => NoComment(request.commentId))
+      // todo NU-1772 only API contract in this MR, BE not adjusted yet
+      scenarioActivityRepository.deleteComment(Random.nextLong())
```

Comment:
This is a incompatible change. Tests should fail for sure. Are we certain that any external system will not fail?
If the answer is yes IMHO we should throw exception to indicate that we're currently changing this API